### PR TITLE
added setting to make author text colored in read only

### PR DESCRIPTION
**Pull request by ookami-kb**
Added setting `markAuthorsReadOnly` to settings. If set to `true` text will be colored by author colors in read-only mode.

### CODE REVIEW

File: src/node/utils/ExportHtml.js [hunk 1]
```diff
@@ -54,7 +55,19 @@ function getPadHTML(pad, revNum, callback)
...
     html = getHTMLFromAtext(pad, atext);
-    callback(null);
+
+    if (markAuthorsReadOnly) {
+      pad.getAllAuthorColors(function(err, authorColors){
+        if(err){
+          return callback(err);
+        }
+
+		html = getHTMLFromAtext(pad, atext, authorColors);
```
Comment (JohnMcLear) [inline src/node/utils/ExportHtml.js]: lulwut indentation..  nope

**Comment by JohnMcLear** [comment]
needs polish

**Comment by JohnMcLear** [comment]
I'm pretty sure this wont work..  Why isn't this just a plugin?

**Comment by Gared** [comment]
@ookami-kb Please fix the indentation

**Comment by muxator** [comment]
It this worth being revived? Needs polish (documentation, adaptation to current code base, entry in `settings.json.template`) but seems interesting overall.

**Comment by JohnMcLear** [comment]
Yea, I mean it's a feature request so it goes on that huge pile we have and one day we will get to :D

**Pull request closed without merging**